packages/vue3/vue.config.js

```diff
@@ -0,0 +1,4 @@
+const { defineConfig } = require("@vue/cli-service");
```

Comment:
are we using this?

Reply:
vue.config.js is automatically loaded by @vue/cli-service, which itself is being used in several npm scripts.